e2e-tests/react-app/FuelConnector.test.ts

```diff
@@ -2,30 +2,91 @@ import { downloadFuel } from '@fuels/playwright-utils';
 import type { FuelWalletTestHelper } from '@fuels/playwright-utils';
 import { test } from '@fuels/playwright-utils';
 import { expect } from '@playwright/test';
-import type { WalletUnlocked } from 'fuels';
-import { bn, toBech32 } from 'fuels';
-import { testSetup } from '../utils/index.js';
+import { type WalletUnlocked, bn } from 'fuels';
+import { testSetup, transferMaxBalance } from '../utils/index.js';
 import { connect } from './utils';
 
 const fuelPathToExtension = await downloadFuel('0.21.0');
```

Comment:
download the latest version always, 0.27.0 is being released right now you can use it

Reply:
Done, but maybe a `latest` tag could be useful in the `downloadFuel` helper.